docs/zh_cn/get_started.md

```diff
@@ -32,15 +32,15 @@ python ${MMDEPLOY_DIR}/tools/deploy.py \
     --dump-info
 ```
 
-`${MMDEPLOY_DIR}/tools/deploy.py` 是一个方便模型转换的工具。可以阅读 [如何转换模型](../en/tutorials/how_to_convert_model.md) 了解更多细节。转换后的模型以及一些其他的信息将会被保存在 `${WORK_DIR}` 中。MMDeploy SDK 可以使用这些信息进行模型推理。
+`${MMDEPLOY_DIR}/tools/deploy.py` 是一个方便模型转换的工具。可以阅读 [如何转换模型](./tutorials/how_to_convert_model.md) 了解更多细节。转换后的模型以及一些其他的信息将会被保存在 `${WORK_DIR}` 中。MMDeploy SDK 可以使用这些信息进行模型推理。
```

Comment:
May change `./tutorials/how_to_convert_model.md` to readthedocs link

Reply:
We have translated this file.